src/View/View.js

```diff
@@ -193,6 +194,7 @@ export default function View({
                         value={searchValue.query}
                       />
                       <Button
+                        aria-label={intl.formatMessage({ id:'stripes-smart-components.search' })}
```

Comment:
`<Button>text</Button>` tag shouldn't have an `aria-label` because `text` will be the default accessible name, e.g. see the `<Button>` immediately below on line 207/209. We should try to figure out why axe is complaining here (maybe just a bad/insufficient mock of `FormattedMessage`?) and resolve it without adding this extra label. 

@JohnC-80 is our a11y expert if you can't get axe to behave for you.